Make Responsive Footer #2

kartikayasija opened the pull request:
closes #2 

Tech Used-
`HTML` `CSS`

Additional Library- 
`Font Awesome`

<img width="960" alt="image" src="https://user-images.githubusercontent.com/115306535/223195840-8651398c-9349-4e7b-b071-a3bb9e9a53f8.png">

AbhiPatel10 commented on index.html:
```diff
@@ -5,7 +5,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
```
Do not use Bootstrap; make it in HTML, CSS, and JavaScript.

Use only the Tech Stack, which is written in the readme file. 

kartikayasija replied:
I have updated the code, check it out.

AbhiPatel10 commented:
Put Company name and Icons on left side

kartikayasija commented:
Updated Footer-
-> Added color to social links hover
-> Added Copyright Section
-> Updated the font family and made the color a little darker


<img width="959" alt="image" src="https://user-images.githubusercontent.com/115306535/223345508-963c3e46-492b-4e6e-b1a7-2320ee2059bb.png">



<img width="958" alt="image" src="https://user-images.githubusercontent.com/115306535/223345443-c87da306-1bf2-4eee-97fc-aa175b7ec825.png">

kartikayasija commented:
> Put Company name and Icons on left side

Okay

kartikayasija commented:
Updated Footer-

- Added colors to social links hover
- Added Copyright section
- Updated company name and icon to left side
- Updated Font family and made the font darker


<img width="960" alt="image" src="https://user-images.githubusercontent.com/115306535/223354129-6f3fd88e-60cd-4095-96ec-f255e6e86f8e.png">



<img width="960" alt="image" src="https://user-images.githubusercontent.com/115306535/223354242-41b74e79-e825-4d3a-bbfb-5359c106ae86.png">

AbhiPatel10 commented on css/style.css:
```diff
@@ -1,9 +1,164 @@
   --primary: #046C5C;
   --secondary: #0F2F51;
 }
+
+a{
+  text-decoration: none;
+}
+
+.container {
+  max-width: 1250px;
+  margin: auto;
+  font-family: 'Poppins', sans-serif;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+ul {
+  list-style: none;
+}
```
```suggestion

a{
  text-decoration: none;
}
ul {
  list-style: none;
}

```

Put ui under a and below ui Comment that footer CSS starting from hear. and end with here.

Do this after that good to go

kartikayasija replied:
Done
